src/renderer/theme.store.ts

```diff
@@ -77,9 +83,9 @@ export class ThemeStore extends Singleton {
     autoBind(this);
 
     // auto-apply active theme
-    reaction(() => this.activeThemeId, async themeId => {
+    reaction(() => this.activeThemeId, themeId => {
       try {
-        this.applyTheme(await this.loadTheme(themeId));
+        this.applyTheme(this.getThemeById(themeId));
```

Comment:
This no longer needs to be `async` because we have a static list. Furthermore, in the future when we support custom themes we can keep this synchronous because the loading of theme files will be done before the call the `applyTheme` is made.